src/context.rs

```diff
@@ -914,11 +952,16 @@ impl SeafowlContext for DefaultSeafowlContext {
         statement: DFStatement,
     ) -> Result<LogicalPlan> {
         // Reload the schema before planning a query
+        // TODO: A couple of possible optimisations here:
+        // 1. Do a visit of the statement AST, and then load the metadata for only the referenced identifiers.
+        // 2. No need to load metadata for the TableProvider implementation maps when instantiating SqlToRel,
+        //    since it's sufficient to have metadata for TableSource implementation in the logical query
+        //    planning phase. We could use a liter structure for that, and implement `ContextProvider` for
```

Comment:
Apologies for bringing up spelling: "liter" -> "lighter"?

Reply:
No worries, fixing 👍🏻